core/src/main/java/hudson/Functions.java

```diff
@@ -650,6 +650,7 @@ public static void configureAutoRefresh(HttpServletRequest request, HttpServletR
             // to avoid conflicts with any other web apps that might be on the same machine.
             c.setPath("/");
             c.setMaxAge(60*60*24*30); // persist it roughly for a month
+            c.setHttpOnly(true); // The HttpOnly flag is a directive to the browser to make sure that the cookie can not be red by malicious script.
```

Comment:
```suggestion
            c.setHttpOnly(true);
```

There is no security impact of this cookie being read or even modified by JavaScript (malicious or not). You only control the fact that the page is automatically refreshed or not.

In addition, this feature will be potentially be removed (see https://github.com/jenkinsci/jenkins/pull/2979 and https://github.com/jenkinsci/jenkins/pull/3894), cc @daniel-beck.